Uchu.World/Objects/GameObjects/Player.cs

```diff
@@ -263,7 +263,11 @@ public override string Name
         /// Whether this player is banned or not
         /// </summary>
         public bool Banned { get; private set; }
-        
+
+        public bool Flying { get; set; } = false;
```

Comment:
Probably better to implement these in some component. You could for example use the destroyablecomponent or something. Currently the player entity already contains too much logic / fields, which isn't really proper ECS.

Reply:
They are already a part of the Controllable physics component and if there are any other variables like this you can encompass all commands in a Slash command component.

Reply:
Ah yes the controllable physics component is indeed the correct place for this.